web-console/src/views/task-view/tasks-view.tsx

```diff
@@ -40,7 +40,7 @@ import { BasicAction } from '../../utils/basic-action';
 
 import './tasks-view.scss';
 
-const supervisorTableColumns: string[] = ['Datasource', 'Type', 'Topic/Stream', 'Status', ActionCell.COLUMN_LABEL];
+const supervisorTableColumns: string[] = ['Datasource', 'Type', 'Topic/Stream', 'Detailed status', ActionCell.COLUMN_LABEL];
```

Comment:
I don't see a reason to change the column name, in the API the new field is `detailedStatus` because `status` is already there but as far as the user is concerned it is the same thing (just more detailed)